android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1996,6 +2003,16 @@ private void InitAssetRatioService() {
         mAssetRatioService = AssetRatioServiceFactory.getInstance().getAssetRatioService(this);
     }
 
+    private void InitPrivacyHubMetrics() {
```

Comment:
nit : method name should in camel case

Reply:
right, i was copying the casing from the methods above this one